frontend/src/pages/GettingStarted.tsx

```diff
@@ -33,25 +33,29 @@ const DEMO_PIPELINES_ID_MAP = {
   data: 2,
   tfx: 1,
   xgboost: 0,
+  v2python: 4,
 };
 
 const PAGE_CONTENT_MD = ({
   control,
   data,
   tfx,
   xgboost,
+  v2python,
 }: {
   control: string;
   data: string;
   tfx: string;
   xgboost: string;
+  v2python: string;
 }) => `
 <br/>
 
 ## Build your own pipeline with
 
-  * TensorFlow Extended (TFX) [SDK](https://www.tensorflow.org/tfx/guide) with end-to-end ML Pipeline Template ([Open a Cloud AI Platform Notebook on Google Cloud](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Ftensorflow%252Ftfx%252Fmaster%252Fdocs%252Ftutorials%252Ftfx%252Ftemplate.ipynb))
   * Kubeflow Pipelines [SDK](https://www.kubeflow.org/docs/pipelines/sdk/)
+    * Check out the latest pipeline support using [SDK v2](https://www.kubeflow.org/docs/components/pipelines/sdk/v2/)
```

Comment:
nit: I'd probably prefer
```
Try out [the new Kubeflow Pipelines SDK v2 (Beta)](https://www.kubeflow.org/docs/components/pipelines/sdk/v2/v2-compatibility/)
```
what do you think?

Reply:
It makes sense, updated the description as suggested.